src/service/apiHelper.js

```diff
@@ -4,86 +4,110 @@ const axios = require('axios');
 const FormData = require('form-data');
 
 class apiHelper {
-  constructor() {}
+  constructor(authHeader) {
+    this.authHeader = authHeader;
+    this.baseHeaders = { Authorization: `${authHeader}` };
+  }
 
-  callToAPIByPost(baseUrl, apiUrl, authHeader, reqBody) {
```

Comment:
Renamed methods:

+ `callToAPIByPost` to `post` 
+ `callToAPIByGet` to `get`, etc.